client/tokenized-express-checkout/transformers/wc-to-stripe.js

```diff
@@ -33,15 +33,15 @@ export const transformPrice = ( price, priceObject ) => {
  * See https://docs.stripe.com/js/appendix/payment_item_object for the data structure
  *
  * @param {Object} cartData Store API Cart response object.
- * @return {{pending: boolean, label: string, amount: integer}} `displayItems` for Stripe.
+ * @return {{pending: boolean, name: string, amount: integer}} `displayItems` for Stripe.
  */
 export const transformCartDataForDisplayItems = ( cartData ) => {
 	const displayItems = cartData.items.map( ( item ) => ( {
 		amount: transformPrice(
 			parseInt( item.prices.price, 10 ),
 			item.prices
 		),
-		label: [
+		name: [
```

Comment:
Slight variation on how the ECE names each line item, compared to PRB: https://docs.stripe.com/js/elements_object/express_checkout_element_shippingaddresschange_event#express_checkout_element_on_shipping_address_change-handler-resolve-lineItems